src/table/Exposure.cc

```diff
@@ -228,6 +228,9 @@ void ExposureFitsWriter::_writeTable(std::shared_ptr<BaseTable const> const &t,
     }
     _mapper = _helper.makeWriteMapper(inTable->getSchema());
     std::shared_ptr<BaseTable> outTable = BaseTable::make(_mapper.getOutputSchema());
+    // Put the metadata from inTable in outTable
+    std::shared_ptr<daf::base::PropertyList> metadata = inTable->getMetadata();
+    outTable->setMetadata(metadata);
```

Comment:
More compact to say simply `outTable->setMetadata(inTable->getMetadata());`, no?
And if you keep the assignment, use `auto`.